### DISCUSSION TITLE
Instantly declined GitHub education application

### DISCUSSION THREAD

**Discussion by kasoob** [community/community#61925] [GitHub Education]
### Hi everyone,

I am seeing the following errors when trying to submit a student GitHub benefits application:

Your application cannot be reviewed until you fix the following:
You are unlikely to be verified until you have completed your GitHub user profile with your full name exactly as it appears in your academic affiliation document plus a short bio. Please do not use a variation of your name or a nickname. Once you have updated your profile information log out and log back into GitHub before re-applying.
If your school does not provide academic email then you must enable use of your device camera to capture your image.

It is very frustrating as I meet these conditions: I do have a full name and bio, and I am using my school's academic email. I have logged out and back in multiple times and tried different browsers, yet it keeps instantly declining my application requests.

**Comment by GhostOf0days** [comment]
Try making a repo,removing the N/A org, and verifying using a screenshot of your cost of attendance bill if it has your school logo on it.

**Reply by kasoob** [reply]
Done all that, still won't work :(

**Reply by GhostOf0days** [reply]
Please create a [Github support ticket](https://support.github.com/).

**Reply by kasoob** [reply]
<img width="417" alt="image" src="https://github.com/community/community/assets/76996113/d405b3d6-9451-469c-a594-511895764989">
It says I can no longer open a ticker for such a request. Perhaps I should choose another option even though I don't think it may entirely apply to me... something like GitHub classroom? I've got no idea how else I can resolve my issue and it seems like I can't contact them regarding applications
